products/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'categories'
+
+class Collection(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'collections'
+
+class Gender(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'genders'
```

Comment:
우식님 젠더의 경우 남녀만 구분할 것이 아니라서 타입으로 작성해주면 더 좋을 것 같습니다!
이에 맞게 Gender를 활용한 코드 전부 수정해주세요!
```suggestion
class GenderType(models.Model):
    type = models.CharField(max_length=45)

    class Meta:
        db_table = 'gender_types'
```